docs/topics/api/overview.rst

```diff
@@ -279,3 +279,4 @@ v4 API changelog
   On addons-dev and addons stage enviroments the previous behavior is available as `api/v4dev`. The `v4dev` api is not available on AMO production server.
   https://github.com/mozilla/addons-server/issues/9467
 * 2018-10-04: added ``is_strict_compatibility_enabled`` to discovery API ``addons.current_version`` object. This change was also backported to the `v3` API. https://github.com/mozilla/addons-server/issues/9520
+* 2018-10-04: added ``exclude_ratings`` parameter to ratings API. This change was also backported to the `v3` API. https://github.com/mozilla/addons-server/issues/9424
```

Comment:
But... we don't need to backport anymore do we? Frontend moved to v4

Reply:
It was more annoying *not* to backport it, since it's just an additional query parameter

Reply:
heh, yeah.  Thought we should consider if our current "one gate per minor feature change" strategy is too heavy for non-breaking changes like this.  Maybe we could have a generic "exclude_v4_rating_features" feature-gate or similar that would .hide all the new stuff we're adding in v4 from v3?